update readme with how to disable autofix (#154)
godaddy/javascript: README.md

}
```

> ## How do I disable auto fix
> 
> You may wish to disable autofix so when you run `eslint` in ci it will error

Reviewer: consistency nit, choose one of the following

```suggestion
## How do I disable autofix

You may wish to disable autofix so when you run `eslint` in ci it will error
```

```suggestion
## How do I disable auto fix

You may wish to disable auto fix so when you run `eslint` in ci it will error
```

> You may wish to disable autofix, so you can run eslint in ci, and error
> if it catches anything unlinted

Reviewer 2: ```suggestion
You may wish to disable autofix so when you run `eslint` in ci it will error
if it catches anything unlinted.
```
